static/app/utils/analytics/issueAnalyticsEvents.tsx

```diff
@@ -109,6 +110,7 @@ export type IssueEventParameters = {
   'issue_details.performance.hidden_spans_expanded': {};
   'issue_details.related_trace_issue.trace_issue_clicked': {
     group_id: number;
+    organization: Organization;
```

Comment:
this shouldn't be necessary and should be required on all analytics, i can look into whats happening